.github/workflows/build-images-action.yml

```diff
@@ -0,0 +1,34 @@
+name: build-images-action
+
+on:
+  push:
+    branches:
+    - 'main'
+    - 'release-*'
```

Comment:
single quotes were removed in main branch, can we keep it consistent ?

Reply:
I'm going to restore them back in main, if this works as expected.